cmd/create/cluster/cmd.go

```diff
@@ -1145,6 +1172,69 @@ func run(cmd *cobra.Command, _ []string) {
 			}
 		}
 	}
+	oidcEndpointUrl := args.oidcEndpointUrl
+	oidcPrivateKeySecretArn := args.oidcPrivateKeySecretArn
+	isByoOidcSet := false
+	if isSTS {
+		// Support byo oidc config
```

Comment:
We can extract lines `1179-1236` to a new function like **handleByoOidcConfig**.